src/main/java/pl/widulinski/webcrawlertool/webDataToScrap/DataToScrap.java

```diff
@@ -0,0 +1,48 @@
+package pl.widulinski.webcrawlertool.webDataToScrap;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.stereotype.Component;
+import pl.widulinski.webcrawlertool.enums.Categories;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+
+
+@Entity
+@Component
+@EqualsAndHashCode
```

Comment:
masz już `@Data`. Natomiast przy encjach zazwyczja lepiej porownywać obiekty po ID poczytaj jak to zrobić w lomboku i czemu to jest ważne